docs/UserGuide.md

```diff
@@ -299,10 +299,13 @@ Switches between different lists in InternConnect stored in `data/` folder.
 Format: `checkout FILE_NAME`
 
 * `FILE_NAME` should not include any extension, as it would always be a JSON file.
-* The command will create a new list populated with the sample data if the specified `FILE_NAME.json` doesn't exist. Otherwise, it will switch to a previously created list.
+* The command will attempt to load the specified `FILE_NAME.json`.
+* If the file contains invalid data, the contents of the file will be erased and an empty addressbook will be loaded.
```

Comment:
Sorry, and one more would be if the file has an invalid format

Reply:
`If the file contains invalid data or have and invalid format, the contents of the file will be erased and an empty 
addressbook will be loaded.`